lib/src/main/java/net/cachapa/expandablelayout/ExpandableLinearLayout.java

```diff
@@ -147,15 +147,23 @@ public boolean isExpanded() {
     }
 
     public void toggle() {
+        toggle(true);
+    }
+
+    public void toggle(boolean animated) {
```

Comment:
Code style: please use "animate" instead of "animated".
This also applies to all other methods